src/TraceEvent/Properties/AssemblyInfo.cs

```diff
@@ -17,3 +17,6 @@
 [assembly: NeutralResourcesLanguage("en")]
 
 // [assembly: InternalsVisibleTo("PerfView")]
+[assembly: AssemblyVersion("1.0.42.0")]
```

Comment:
I would like to follow the simplest/most commonly used approach (to NOT be innovative).   Ideally humans don't deal with version numbers much at all (only on major transitions).